libpod/events.go

```diff
@@ -0,0 +1,81 @@
+package libpod
+
+import (
+	"github.com/containers/libpod/libpod/events"
+	"github.com/hpcloud/tail"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+// newContainerEvent creates a new event based on a container
+func (c *Container) newContainerEvent(status events.Status) {
+	e := events.NewEvent(status)
```

Comment:
Shouldn't events.NewEvent just take these fields as  paramaters?

Reply:
this is way more convenient for adding the events later. and most of it can be derived from the method object (container, pod...)